backend/Api/Forecasts/ForecastController.cs

```diff
@@ -0,0 +1,65 @@
+using Api.Common;
+using Api.Helpers;
+using Core.Consultants;
+using Core.Extensions;
+using Core.Forecasts;
+using Core.PlannedAbsences;
+using Core.Staffings;
+using Infrastructure.DatabaseContext;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Caching.Memory;
+
+namespace Api.Forecasts;
+
+[Authorize]
+[Route("/v0/{orgUrlKey}/forecasts")]
+[ApiController]
+public class ForecastController(
+	ApplicationContext context,
+	IMemoryCache cache,
+	IForecastRepository forecastRepository,
+	IStaffingRepository staffingRepository,
+	IPlannedAbsenceRepository plannedAbsenceRepository)
+	: ControllerBase
+{
+	[HttpGet]
+	public async Task<ActionResult> Get(
+		[FromRoute] string orgUrlKey,
+		CancellationToken cancellationToken,
+		[FromQuery(Name = "Date")] DateOnly? requestedDate = null,
+		[FromQuery(Name = "MonthCount")] int monthCount = 0)
```

Comment:
```suggestion
		[FromQuery(Name = "MonthCount")] int monthCount = 12)
```

Is a default value of 12 better here?

Reply:
Yeah!